src/SamenwerkendeCatalogi/Models/ScItem.php

```diff
@@ -58,26 +58,26 @@ private function assignDoelgroepen($doelgroepen = [], $doelgroepTerm): array
         return $doelgroepen;
     }
 
-    /**
-     * @return string
-     */
-    public function getUplName(): string
+    public function getUplName(bool $strip = false): string
```

Comment:
Ik ben nooit zo'n fan van boolean parameters die het gedrag van een method veranderen. Je zou in plaats daarvan een losse method kunnen toevoegen, bijv. `getStrippedUplName()`. Dit geeft duidelijk aan wat de method doet. 

Stel je komt ergens in de code dit tegen: `getUplName(false)`. Nu kun je eerst de method zelf opzoeken om er achter te komen wat die `false` ook al weer doet. Stript 'ie hem of juist niet? :)